README.md

```diff
@@ -6,11 +6,7 @@ This plugin enables project admins to export project member data.
 
 ## Requirements
 
-This plugin **requires PostgreSQL/PostGIS** and depends on the [`redmine_gtt` Plugin](https://github.com/gtt-project/redmine_gtt)!!!
-
-- Redmine >= 3.4.0
-- PostgreSQL >= 9.6
-- PostGIS >= 2.4
```

Comment:
@dkastl I deleted GTT plugin dependency, because actually, there is no GTT plugin related part, although plugin name includes `gtt`.